pkg/exchange/binance/stream.go

```diff
@@ -251,6 +502,10 @@ func (s *Stream) dial(listenKey string) (*websocket.Conn, error) {
 		url = "wss://stream.binance.com:9443/ws/" + listenKey
 	}
 
+	if s.IsFutures {
+		url = "wss://fstream.binance.com/ws/" + listenKey
+	} 
```

Comment:
it's possible to use (futures + public), can you pull out this to a method? like `getEndpointURL()`?